src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/SourceEditModePageObject.java

```diff
@@ -391,13 +401,34 @@ public ArticlePageObject clickPublishButton() {
     return new ArticlePageObject(driver);
   }
 
-  public void addCategoryToSourceCode(String catName) {
-    sourceModeTextArea.sendKeys(catName);
-  }
-
   public TemplatePageObject clickPublishButtonInTemplateNamespace() {
     wait.forElementVisible(submitButton);
     submitButton.click();
     return new TemplatePageObject(driver);
   }
+
+  public SourceEditModePageObject addContentToTextArea(String text) {
+    wait.forElementVisible(textAreaScroller);
+    new Actions(driver)
+        .click()
+        .sendKeys(text)
+        .build().perform();
+    return this;
+  }
+
+  public SourceEditModePageObject deleteCharacter(Integer howMany) {
+    wait.forElementVisible(textAreaScroller);
+    Actions action = new Actions(driver);
+    action
+        .moveToElement(textAreaScroller)
+        .click();
+    for (int i = 0; i < howMany; i++)
+    {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this left curly brace to the end of previous line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ALeftCurlyBraceEndLineCheck)